starknet_py/net/account/account.py

```diff
@@ -86,6 +86,10 @@ def __init__(
         self._address = parse_address(address)
         self._client = client
         self._cairo_version = None
+        if isinstance(chain, str):
+            self._chain_id = int(chain, 16)
```

Comment:
I am not sure if we should expect a hex string here. Allowing a `str` to be passed suggests that the user can for example pass `SN_GOERLI` and in my opinion we should treat it that way.

Reply:
The main reason why I want to do that is that our method `get_chain_id` returns `hex string` and we also want to support strings for example "KATANA". Even if someone passes `SN_GOERLI` it will work. I know it isn't the best solution, but we have some examples in docs on how to use it correctly with enym

Reply:
In the current solution if a string is passed, for example `chain="KATANA"` then it will throw an error `ValueError: invalid literal for int() with base 16: 'KATANA'`.

Reply:
Thx for catch it! I pushed fix